index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en"> 
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@100;200&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+     <img  class="alignleft" src="image_geometry_1.svg" alt="header image">
+     <img  class="alignright" src="ic_menu.svg" alt="ic-menu image">
+
+     
+    </header>
+
+    <section> 
+    <h1 class="alignleft">Charlie Giorgania</h1>
+    <p id="headline" class="alignleft" >Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+    <div id="box">Start collaboration</div>
+    </section>
+
+    <footer>
+        <img  src="image_geometry_2.svg" alt="footer image">
```

Comment:
- [ ] Kindly remove this image from HTML and add it as `background-image` in CSS as this is part of the design and it's not an actual HTML element 👍

Reply:
Fixed.